### PR TITLE
Fix that qr scanner doesn't work correctly for 'To' field

### PR DESCRIPTION
**Pull request by Nasicus**
Problem was that `ng-model` is not supported on `md-autocomplete`.

To work around this we have to set the `selectAddress` property which is used on the `md-autocomplete`. However we cannot really set the field directly from the qr scanner directive (because it a "contact").

My solution was to introduce a callback-function parameter (which I personally find nicer anyway, instead of having to relay on, that the object path `send.data` exists).

Please note that I'm not very good with angular 1 - and it may very be possible that there is a better fix for this. I'm always open for feedback :)

fixes: #454

### CODE REVIEW

File: client/app/src/components/qr-scanner/qr-scanner.directive.js [hunk 1]
```diff
@@ -63,7 +69,8 @@
         onSuccess: '&',
         onError: '&',
         onVideoError: '&',
-        inputCallback: '@'
+        inputCallback: '@',
+        inputCallbackFunc: '&'
```
Comment (alexbarnsley) [inline client/app/src/components/qr-scanner/qr-scanner.directive.js]: From what I can tell, this isn't used. Is that correct @Nasicus

**Comment by alexbarnsley** [comment]
Just 1 question/comment really. Tested and it's working 👍

File: client/app/src/components/qr-scanner/qr-scanner.directive.js (hunk 1 above)
Comment (Nasicus) [reply client/app/src/components/qr-scanner/qr-scanner.directive.js]: I'm not sure I understand: I'ts caled here: https://github.com/ArkEcosystem/ark-desktop/pull/456/files#diff-3671ac06b4eabc94ab421779e16417c1R28 (Line 28) and the "value" is set here: https://github.com/ArkEcosystem/ark-desktop/pull/456/files#diff-9ce1e6b4b80b3cd9b73da170356b437cR17

Comment (alexbarnsley) [reply client/app/src/components/qr-scanner/qr-scanner.directive.js]: My apologies, I was being thick! 👌

**Comment by alexbarnsley** [comment]
Nice work! +5! 👍

**Pull request merged**